frameworks/digital-outcomes-and-specialists/questions/services/businessAnalystLocations.yml

```diff
@@ -0,0 +1,16 @@
+hint: ''
```

Comment:
Don't think it'll cause any problems but it's sort of unclear what the intention of a blank `hint` is.